charts/replicated-library/values.yaml

```diff
@@ -43,11 +43,14 @@ apps:
      # -- Specify the controller type.
      # Valid options are deployment, daemonset or statefulset
      # TODO: daemonset and statefulset
+     # When deployment type: statefulset:
```

Comment:
maybe this should be "app" type instead of "deployment" type to avoid confusion?

Reply:
Yup I agree, in fact now that I'm looking at that I'm going to propose we remove the comments all together. There's not enough room here to fully explain the differences in deployments and statefulsets and I'm concerned the hints that were left here may actually lead to more confusion rather than less.